funnel/translations/messages.pot

```diff
@@ -3831,15 +3845,15 @@ msgid ""
 "comment</a>."
 msgstr ""
 
-#: funnel/templates/notifications/layout_email.html.jinja2:39
+#: funnel/templates/notifications/layout_email.html.jinja2:35
 msgid "Unsubscribe or manage preferences"
 msgstr ""
 
-#: funnel/templates/notifications/layout_web.html.jinja2:7
+#: funnel/templates/notifications/layout_web.html.jinja2:6
 msgid "(This template is missing content)"
 msgstr ""
 
-#: funnel/templates/notifications/macros_email.html.jinja2:6
+#: funnel/templates/notifications/macros_email.html.jinja2:5
 #, python-format
 msgid "Update #%(number)s &bull; Posted by %(editor)s %(age)s"
```

Comment:
This should be two separate trans tags. The `&bull;` is markup and not translatable.